package/MDAnalysis/lib/formats/libmdaxdr.pyx

```diff
@@ -306,7 +306,8 @@ cdef class _XDRFile:
 
         # where was I
         current_frame = state[1]
-        self.seek(current_frame)
+        self.seek(current_frame - 1)
+        self.current_frame = current_frame
```

Comment:
Do these lines make the low-level tests fail?

Reply:
The current tests can pass if these changes are ditched, but still the last frame cannot be easily pickled as  `seek` here is sort of different from that in DCD files.